modules/wolfgame.py

```diff
@@ -463,7 +466,7 @@ def pinger(cli, nick, chan, rest):
     var.PINGING = True
     TO_PING = []
 
-
+    pl = len(var.list_players())
```

Comment:
This should be moved into on_whoreply to account for cases where someone does !ping and someone else joins before the WHO reply comes in -- that way the number of players can always be accurate.

Also, don't use pl as the var name, normally that name stands for "player list" aka the actual list of players, not the length of the player list; pick a different name so we aren't overloading the semantic meaning of variable names.